### DISCUSSION TITLE
How to use Skywalking in GraalVm

### DISCUSSION THREAD

**Discussion by xiaozaiyuji** [apache/skywalking#10322] [Q&A]
I was test the combination Skywalking Java Agent and GraalVm.But I did not know how to start,except the the doc https://www.graalvm.org/latest/docs/introduction/ and https://docs.spring.io/spring-boot/docs/current/reference/html/native-image.html#native-image.So can any body give me a experience or advice.

**Comment by Superskyyy** [comment]
@xiaozaiyuji  https://github.com/apache/skywalking/issues/2573 Check this, it should work already? Even the jvm info was collected, at least back then
